spec/facter/resolvers/linux/networking_spec.rb

```diff
@@ -27,6 +27,10 @@
         .with('/sys/class/net/lo/operstate', nil).and_return('unknown')
       allow(Facter::Util::FileHelper).to receive(:safe_read)
         .with('/sys/class/net/ens160/operstate', nil).and_return('up')
+      allow(Facter::Util::FileHelper).to receive(:safe_read)
+        .with('/sys/class/net/ens160/speed', nil).and_return(1000)
+      allow(Facter::Util::FileHelper).to receive(:safe_read)
+        .with('/sys/class/net/ens160/duplex').and_return('full')
```

Comment:
This fixes some of the issues:

```suggestion
        .with('/sys/class/net/ens160/duplex', nil).and_return('full')
```

Reply:
Fixed.